src/AttackDisplay.cpp

```diff
@@ -52,7 +52,7 @@ void AttackDisplay::Init( const PlayerState* pPlayerState )
 			attacks.insert( asAttacks[att] );
 	}
 
-	for( set<RString>::const_iterator it = attacks.begin(); it != attacks.end(); ++it )
+	for( auto it = attacks.begin(); it != attacks.end(); ++it )
```

Comment:
If you're going to use `auto` here, you should change this to `cbegin()` and `cend()` to retain the const-correctness that the old version has.